meeting_minutes/220616_SARIF_TC_60.md

```diff
@@ -0,0 +1,177 @@
+
+### 1.7.1 Prospective members attending their first meeting
+
+* Chris Wysopal will obtain voting rights at the end of the next meeting.
+* Johnathan Gilday will obtain voting rights at the end of the next meeting.
+* Pavel Mikula will obtain voting rights at the end of the next meeting.
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+* Mary Martin will obtain voting rights at the end of this meeting.
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Co-Chair Keaton)
+
+- Scheduled Teleconferences (Thursdays at 08:00 PT / 15:00 UTC for 1.5 hours)
+
+    ```
+    June 30
+    ```
+- Proposed Teleconferences (Thursdays at 08:00 PT / 15:00 UTC for 1.5 hours)
+
+    ```
+    July 14
+    July 28
+    ```
+- Possible face-to-face meeting when pandemic permits
+
+# 3. Liaisons
+
+## 3.1 OpenSSF
+
+### 3.1.1 Choose who will act as liaison
+
+* Thanassis: Volunteers to act as liaison
+* Jeff: Possibly Contrast Security might share some liaison duties
+* Michael: Microsoft may assist taking over liaison duties when shared with others 
+* David: Proposal is that the Liaison Group is: Thanassis, Contrast Security, and Microsoft
+  * Motion: So moved, seconded, and approved with unanimous consent.
+### 3.1.2 Arrange to initiate contact with OpenSSF
+
+* David: Next step is to contact the Security Tools OpenSSF working group
+* Michael: Will kindly request assistance from Microsoft contact to speed up contact and initiation.
+
+# 4. Discussion
+
+## 4.1 Review recruitment effort to complete the technical committee
+
+* No news
+* David: Proposes to remove this item from future agendas
+* No objection all agree
+## 4.2 Review status on finalizing SARIF 2.1 errata [List of next steps - github issue #509](https://github.com/oasis-tcs/sarif-spec/issues/509)
+
+* Michael: Will ensure the remaining tasks noted on ticket at github are completed and then sends back to David for completion of submittal to OASIS for progression
+
+## 4.3 Review current state of ecosystem ongoing work
+
+* Mary: SARIF integration for various linters ongoing (PRs in progress) some already accepted by the repository owners
+* Michael: Github has interest and thus has helpfully contacted some of those respective repository owners to accelerate the SARIF format capabilities in use there
+* Michael: Progress has been made at Microsoft on tracking code locations for results as the developers are making edits 
+* Michael: Internal discussions at Microsoft for inclusion of SARIF shared components into native Mircosoft tools (esp. C++)
+* Michael: Other progress will be shown in the demo by Mary later in the meeting
+## 4.4 Any continued report/discussion on metrics
```

Comment:
>metrics [](http://example.com/codeflow?start=42&length=7)

I think we can remove this agenda item for now, as it isn't our current focus and we haven't had any status for at least three meetings.

Reply:
Sounds good.  I will remove it from the next meeting's agenda.